### DISCUSSION TITLE
Updated with comfyui manager and now comfyui doesnt start

### DISCUSSION THREAD

**Discussion by Ai-Enrapture** [Comfy-Org/ComfyUI-Manager#136] [General]
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\SD ComfyUI\\ComfyUI\\custom_nodes\\.git\\__init__.py'

Cannot import C:\SD ComfyUI\ComfyUI\custom_nodes\.git module for custom nodes: [Errno 2] No such file or directory: 'C:\\SD ComfyUI\\ComfyUI\\custom_nodes\\.git\\__init__.py'

why does an addon that is supposed to be helpful does the opposite?

**Comment by ltdrdata** [comment]
Something went wrong. There shouldn't be a `.git` directory in the `custom_nodes` dir. 

It's strange that it appeared in that location when you update because manager doesn't perform `git clone` for updates. Have you manually extracted any files into the `custom_nodes` directory?

If that's not the case, it means that for a specific custom node, the install script accidentally received an incorrect git clone.

I need to identify which custom node might have caused the issue, so please share the contents of the `custom_nodes/.git/config` file here, and delete the `custom_nodes/.git` directory.

**Comment by ltdrdata** [comment]
I have previously reproduced and patched a similar issue and patched. Perhaps it won't occur now.
